dcpy/utils/postgres.py

```diff
@@ -5,20 +5,23 @@
 import pandas as pd
 from psycopg2.extensions import AsIs
 from sqlalchemy import create_engine, text
+from typing import Optional
 
 
 BUILD_ENGINE_RAW = os.environ["BUILD_ENGINE"]
 build_engine = create_engine(BUILD_ENGINE_RAW)
 
 
-def execute_query(query: str, placeholders: dict = None) -> None:
+def execute_query(query: str, placeholders: Optional[dict] = None) -> None:
     if placeholders is None:
         placeholders = {}
     with build_engine.connect() as connection:
-        connection.execute(statement=text(query), params=placeholders)
+        connection.execute(statement=text(query), parameters=placeholders)
```

Comment:
FYI, this was one of the bugs. I haven't tested usages of this fn. We could benefit from some integration testing here!